core/src/display_object/edit_text.rs

```diff
@@ -803,6 +803,9 @@ impl<'gc> EditText<'gc> {
             if !is_word_wrap {
                 edit_text.bounds.set_x(new_x);
                 edit_text.bounds.set_width(width);
+            } else {
+                let max_width = edit_text.static_data.text.bounds.x_max;
```

Comment:
Hmm, why not `max - min`?

Reply:
The width and position of the text field shouldn't change at all if word wrap is on, only the height -- [AS3 docs](https://help.adobe.com/en_US/FlashPlatform/reference/actionscript/3/flash/text/TextField.html#autoSize):

> If wordWrap is also set to true, only the bottom of the text field is resized and the right side remains fixed.

Reply:
Scratch that: per the below discussion, setting the width to `x_max - x_min` is a good solution for now. Let's add a TODO comment that eventually the width shouldn't change.